src/core/request.js

```diff
@@ -58,8 +58,10 @@ const logger = loggerFactory('core/request');
  */
 const createError = (response, fallbackMessage, httpCode, httpSent) => {
     let err;
-    if (response && response.errorCode) {
-        err = new Error(`${response.errorCode} : ${response.errorMsg || response.errorMessage || response.error}`);
+    const code = response && (response.errorCode || response.code);
```

Comment:
Since `code` is the expected standard we have it always as first check :) 
 
```suggestion
    const code = response && (response.code || response.errorCode);
```